src/Enum/OrderIncludes.php

```diff
@@ -4,11 +4,12 @@
 
 enum OrderIncludes: string
 {
-    case BILLING   = 'billing';
-    case COMMENTS  = 'comments';
-    case DISCOUNTS = 'discounts';
-    case ITEMS     = 'items';
-    case PAYMENT   = 'payment';
-    case TAGS      = 'tags';
-    case TAXES     = 'taxes';
+    case BILLING        = 'billing';
+    case COMMENTS       = 'comments';
+    case DISCOUNTS      = 'discounts';
+    case ITEMS          = 'items';
+    case PAYMENT        = 'payment';
+    case TAGS           = 'tags';
+    case TAXES          = 'taxes';
+    case CUSTOM_FIELDS  = 'custom_fields';
```

Comment:
Even in alfabetische volgorde zetten

Reply:
Yes is goed, thanks alvast, hou er wel rekening mee dat de PR nog in draft staat, moet nog best wat dingen doen